spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -14,10 +14,6 @@ management:
           autotime:
             enabled: true # true is default to Boot 2.3.2 at least.
             percentiles-histogram: true
-    export:
```

Comment:
Could you help me understand why these properties are being removed and the addition of `metrics.export.rsocket.enabled`?

Reply:
It was a change in SB 3.x that facilitated the need for this change: https://github.com/spring-projects/spring-boot/commit/be3523b1cda41f178df25fdfe8f8f27199259530

Reply:
See https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#actuator-metrics-export-properties

Reply:
The RSocket property was a miss when we bumped to prometheus-rsocket-proxy 2.0.0-M1 (see 2nd half of PR description).